homeassistant/components/telegram_bot/strings.json

```diff
@@ -636,5 +636,15 @@
         }
       }
     }
+  },
+  "issues": {
+    "proxy_params_auth_deprecation": {
+      "title": "Proxy authentication should be moved to the URL",
```

Comment:
Mayne start with the integration name in the title? Add a translation placeholder for it, since it's a name that shouldn't be translated.